homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -45,13 +46,13 @@ class AdvantageAirFilter(AdvantageAirAcEntity, BinarySensorEntity):
     _attr_entity_category = EntityCategory.DIAGNOSTIC
     _attr_name = "Filter"
 
-    def __init__(self, instance, ac_key):
+    def __init__(self, instance, ac_key: str) -> None:
```

Comment:
For now, `instance` should be `dict[str, Any]` (multiple locations where this can be applied)
```suggestion
    def __init__(self, instance: dict[str, Any], ac_key: str) -> None:
```